internal/versions/versions.go

```diff
@@ -116,3 +119,20 @@ func (h *handler) GetSupportedVersionFormat(openshiftVersion string) (string, er
 	}
 	return fmt.Sprintf("%d.%d", v.Segments()[0], v.Segments()[1]), nil
 }
+
+func (h *handler) getOpenshiftVersion(openshiftVersion string) *models.OpenshiftVersion {
+	if v, ok := h.openshiftVersions[openshiftVersion]; ok {
+		return &v
+	}
+
+	if h.db != nil {
+		var v models.OpenshiftVersion
+		if err := h.db.First(&v, "display_name = ?", openshiftVersion).Error; err != nil {
+			// The specified openshift version is not supported
+			return nil
+		}
+		return &v
+	}
```

Comment:
I'm a little bit confused about how this works. Does it look for only customized versions in the DB?

In addition, `openshiftVersion` is the key of the JSON (currently x.y - 4.6, 4.7)... The `display_name` is a textual representation for the users when they have to choose from the dropdown list.

Reply:
removed db handling in current impl